includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -6,6 +6,14 @@ class DesignSystemApiController extends WikiaApiController {
 	const PARAM_LANG = 'lang';
 	const PRODUCT_WIKIS = 'wikis';
 
+	protected $cors;
+
+	public function __construct() {
+		parent::__construct();
+		$this->cors = new CrossOriginResourceSharingHeaderHelper();
+		$this->cors->setAllowOrigin( [ '*' ] );
```

Comment:
I wonder if this should be more specific. 

There's probably no harm in having this be open to everyone, but maybe we should be a little paranoid with this sort of thing and restrict it to `*.wikia.com` and `*.wikia-dev.pl`, `*.wikia-dev.us` for now. What do you think?